code/__DEFINES/reagents.dm

```diff
@@ -0,0 +1,334 @@
+//Reagents
+
```

Comment:
Even though this is going to give me a bunch of conflicts, it is so much better in the long run.